rocks-db/src/migrations/clean_update_authorities.rs

```diff
@@ -23,9 +24,14 @@ impl From<AssetCollectionBeforeCleanUp> for AssetCollection {
     }
 }
 
+impl<'a> ToFlatbuffersConverter<'a> for AssetCollection {
```

Comment:
So this implementation will panic on runtime, is that expected

Reply:
Wdym? It will not. `AssetCollection` doesn't do calls to flatbuff methods. In case it did, it would panic.